src/aliases/store/postgres/alias.go

```diff
@@ -0,0 +1,58 @@
+package aliaspg
+
+import (
+	"context"
+	goerrors "errors"
+
+	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	models "github.com/consensys/quorum-key-manager/src/aliases/store/models"
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
+)
+
+var _ aliasstore.Alias = &Alias{}
+
+// Alias stores the alias data in a postgres DB.
+type Alias struct {
+	pgClient postgres.Client
+}
+
+func NewAlias(pgClient postgres.Client) *Alias {
+	return &Alias{
+		pgClient: pgClient,
+	}
+}
+
+func (s *Alias) CreateAlias(ctx context.Context, registryName models.RegistryName, alias models.Alias) error {
+	alias.RegistryName = registryName
+	err := s.pgClient.Insert(ctx, &alias)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (s *Alias) GetAlias(ctx context.Context, registryName models.RegistryName, aliasKey models.AliasKey) (*models.Alias, error) {
+	a := models.Alias{Key: aliasKey, RegistryName: registryName}
+	err := s.pgClient.SelectPK(ctx, &a)
+	return &a, err
+}
+
+func (s *Alias) UpdateAlias(ctx context.Context, registryName models.RegistryName, alias models.Alias) error {
```

Comment:
For consistency, I would suggest to return also the the update object as we do in other DB implementations (see below). I also that is a more standard practice
```
Update(ctx context.Context, account *entities.ETH1Account) (*entities.ETH1Account, error) {
```

Reply:
I'm ok with it.